[ADD][FIX] Multiple changes

zaoral opened the pull request:
This is required since create new records without company will make then
invisible, odoo will search for the ones with the last date with company
set. If a user create a rate without company this one will never be take into
account. Adding the constraint we avoid this human error.

zaoral committed:
[ADD] account_ux: do not let the user create rates without company if already
had rates with company before the new rate date

This is required since create new records without company will make then
invisible, odoo will search for the ones with the last date with company
set. If a user create a rate without company this one will never be take into
account. Adding the constraint we avoid this human error.

jjscarafia commented on account_ux/models/res_currency_rate.py:
```diff
@@ -0,0 +1,25 @@
+        for rec in self.filtered(lambda x: not x.company_id):
+            others_with_company = self.search([
+                ('name', '<=', rec.name),
+                ('currency_id', '<=', rec.currency_id.id),
```
Acá imagino que sería = en vez de <=, no?

zaoral replied:
@jjscarafia  puse >= porque en el caso particular de las res.currency.rate el campo name es de tipo fecha. nos interesa son los rates que sean en fecha previa o igual a la fecha del nuevo rate.

jjscarafia replied:
perfecto, eso lo vi, pero el <= está tmb en el currency "'currency_id', '<=', rec.currency_id.id"
Me refería a ese pero no quedó claro...

zaoral replied:
Tienes razon Juan, no lo habia visto. voy a corregirlo

zaoral replied:
Listo el cambio y el push

jjscarafia reviewed (changes requested):
Excelente kathy, dejo un comentario por una pavada..

zaoral committed:
[FIX] account_debt_management: Use Financial Amounts was overlapping Update
Currency Live option.

Fixed by updating the id for the new div for Use Financial Amounts section and
adding a difference text that does match with the option purpose.
